dev-libs/libprelude/libprelude-5.2.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+USE_RUBY="ruby24 ruby25 ruby26 ruby27"
```

Comment:
Ruby-2.4 doesn't exist in the tree anymore.

Reply:
Done